teams/Aragon One/2019.md

```diff
@@ -57,9 +56,7 @@ The aim of this initiative is to increase the richness and flexibility of the ex
 
 Focus areas for this initiative:
 
-- **Stateful permissions**: Allow users to set and manage time-based and use-based permissions.
```

Comment:
Reason to remove: We won't have design nor frontend bandwidth to think or implement it during Q4 2019. Our initial research concludes that implementing something like this can imply almost building a graphical programming language with if/else/etc. statements, which would make terrible UX and also be terribly hard to implement.